db/migrate/20221017124304_add_user_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Please change the user reference column name to `author` instead of `user`, as per [the given ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).